### PR TITLE
Add public_cert support

### PR DESCRIPTION
**Pull request by nitzann**
https://github.com/external-secrets/external-secrets/issues/847

### CODE REVIEW

File: docs/provider-ibm-secrets-manager.md
Comment (knelasevero) [inline docs/provider-ibm-secrets-manager.md:52]:
```suggestion
We support the following secret types of [IBM Secrets Manager](https://cloud.ibm.com/apidocs/secrets-manager): `arbitrary`, `username_password`, `iam_credentials`, `public_cert` and `imported_cert`. To define the type of secret you would like to sync you need to prefix the secret id with the desired type. If the secret type is not specified it is defaulted to `arbitrary`:
```

**Comment by knelasevero** [comment]
Thanks @nitzann! Just a small typo there :)

This LGTM but I don't have an IBM account to test anymore (and e2e for IBM are not running on the projects infra).

**Review by IdanAdar — APPROVED** [review]

**Comment by IdanAdar** [comment]
@knelasevero anything else needed before sign off?

**Comment by knelasevero** [comment]
Gh actions is having a hard time today it seems. Gonna merge tomorrow :)

**Comment by knelasevero** [comment]
Closed and re-opened to trigger CI, after that we can merge.

BTW, noticed that you 2 work for IBM, would it be ok to add you as maintainers of this specific provider and ask you for help when we need to test something?

**Comment by IdanAdar** [comment]
Thank you! Do you know when is the next cut off for a new version release?

**Comment by moolen** [comment]
We've had a lot of substantial changes since `0.4.4`. I hope we get everything in the next two weeks and prep a release afterwards. If it is urgent for you let us know then we can consider rebasing it on top of `0.4.4` in case we hit a roadblock that defers the next release for a bit.

**Comment by IdanAdar** [comment]
@moolen No urgency, but would like the see the support for `kv` and `public_cert` secret types available for consumption... Do you think we're getting there?

**Comment by moolen** [comment]
yes, answered here: https://github.com/external-secrets/external-secrets/pull/870#issuecomment-1087853846

**Pull request merged**
